docs/reference/modules/http.md

```diff
@@ -0,0 +1,186 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/http`
+
+**Alias:** `apos.http`
+
+This module adds methods for easy HTTP requests in Apostrophe project and module server-side code. It also defines error codes for use with the main `apos.error()` method.
```

Comment:
It should be mentioned here that these methods are deliberately compatible with those available on the browser side for developer convenience.